downstream/modules/platform/proc-operator-aap-faq.adoc

```diff
@@ -10,7 +10,7 @@ You can follow along in the command line while the operator is reconciling, this
 Alternatively you can click into the deployment instance to see the status conditions being updated as the deployment goes on.
 
 Is it still possible to view individual component logs?::
-When troubleshooting you should examine the *AnsibleAutomationPlatform* instance for the main logs and then each individual component (*EDA*, *AutomationHub*, *AutomationController*) for more specific information. 
+When troubleshooting you should examine the *Ansible Automation Platform* instance for the main logs and then each individual component (*EDA*, *AutomationHub*, *AutomationController*) for more specific information. 
```

Comment:
Examine the {PlatformNameShort} instance main logs and then individual components () when troubleshooting for more specific information.

'each individual' says the same thing twice